app/resources/tos.html.template

```diff
@@ -1,14 +1,40 @@
-{# Terms of Service page for a new site.  Fill in content below.            #}
+{# Copyright 2015 Google Inc.  Licensed under the Apache License, Version   #}
+{# 2.0 (the "License"); you may not use this file except in compliance with #}
+{# the License.  You may obtain a copy of the License at:                   #}
+{#     http://www.apache.org/licenses/LICENSE-2.0                           #}
+{# Unless required by applicable law or agreed to in writing, software      #}
+{# distributed under the License is distributed on an "AS IS" BASIS,        #}
+{# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. #}
+{# See the License for the specific language governing permissions and      #}
+{# limitations under the License.                                           #}
+
+{# Template for the Terms of Service page.                                 #}
 
 {% extends "static-base.html.template" %}
+{% load i18n %}
 
 {% block content %}
-  <h1>Terms of Service</h1>
-
-  <p>
-  (This page has not been filled in by the site owner.
-  It is recommended that site owners specify their Terms of Service
-  when launching a Person Finder site.  Site owners should see the
-  <a href="https://github.com/google/personfinder">project page</a>
-  for instructions on filling in the content of this page.)
-{% endblock content %}
+
+  {% if article_html %}
+
+    {{article_html|safe}}
+
+  {% else %}{% if tos_url %}
+
+    <h1>Terms of Service</h1>
+
+    {# In case we fail to fetch/parse the ToS page (e.g., the page has       #}
+    {# changed its layout), we fall back to show a link to the page instead. #}
+    <p><a href="{{tos_url}}">Click here to see the terms of service (external site)</a></p>
```

Comment:
It may require payment, so I think it's better not to redirect automatically. What do you think?